libraries/chain/controller.cpp

```diff
@@ -2956,6 +2977,24 @@ void controller_impl::on_activation<builtin_protocol_feature_t::preactivate_feat
    } );
 }
 
+template<>
+void controller_impl::on_activation<builtin_protocol_feature_t::replace_deferred>() {
+   const auto& indx = db.get_index<account_ram_correction_index, by_id>();
+   auto itr = indx.begin();
```

Comment:
Looks like this line ```auto itr = indx.begin();``` is redundant and "itr" will be hided by another "itr" in the for loop.